CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replaced hard-coded color values. [skip-release-notes]
 - Added a toggle for “Enable new media display” to Staging builds. [skip-release-notes]
 - Fixed typos in release notes. [skip-release-notes]
+- Localized the quotation marks on the Notifications view.
```

Comment:
I think I'd like to have these sorted, so the ones that go in the release notes are above all the ones that say `[skip-release-notes]`. Would it make more sense to just have a section like `### Internal` under `## [Unreleased]`, and then it's implied that the Internal ones are skipped? Or two sections: `### External` and `### Internal`? Or maybe even better names than those -- I'm just not coming up with anything else right now.

This is just an idea; I don't think we need to do it right now unless we settle on a good approach, but I wanted to start talking about it.